projects/plugins/jetpack/extensions/blocks/donations/index.js

```diff
@@ -24,7 +24,38 @@ export const settings = {
 	description: __( 'Collect one-time, monthly, or annually recurring donations.', 'jetpack' ),
 	icon: DonationsIcon,
 	category: 'earn',
-	keywords: [ __( 'Donations', 'jetpack' ) ],
+	keywords: [
+		_x( 'charity', 'block search term', 'jetpack' ),
+		_x( 'contribution', 'block search term', 'jetpack' ),
+		_x( 'commerce', 'block search term', 'jetpack' ),
+		_x( 'credit card', 'block search term', 'jetpack' ),
+		_x( 'debit card', 'block search term', 'jetpack' ),
+		_x( 'donate', 'block search term', 'jetpack' ),
+		_x( 'Donations', 'block search term', 'jetpack' ),
+		_x( 'earn', 'block search term', 'jetpack' ),
+		_x( 'ecommerce', 'block search term', 'jetpack' ),
+		_x( 'fundraising', 'block search term', 'jetpack' ),
+		_x( 'fundraiser', 'block search term', 'jetpack' ),
+		'gofundme',
+		'go fund me',
```

Comment:
This one should be removed, it'd match individual generic words and doesn't make sense